### PR TITLE
Update core-committers.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Added Adolfo to core committers.

### CODE REVIEW

File: contributors/contributors/core-committers.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ A core committer is a maintainer on the Mattermost project that has merge access
 
 Below is the list of core committers working on Mattermost:
 
-* [**Angelos Kyratzakos**](core-committers.md)
-  * @angelos.kyratzakos on [community.mattermost.com](https://community.mattermost.com/core/messages/@angelos.kyratzakos) and [@angeloskyratzakos](https://github.com/angeloskyratzakos) on GitHub
-  * Dev areas: Kubernetes, Infrastructure
+* [**Adolfo García Veytia**](core-committers.md)
+  * @puerco on [community.mattermost.com](https://community.mattermost.com/core/messages/@puerco) and [@puerco](https://github.com/puerco) on GitHub
+  * Dev areas: DevOps
```
Comment (cpanato) [inline contributors/contributors/core-committers.md]: I would suggest adding `SBOM`, `Containers`, `Kubernetes`, `Supply Chain`

anything else you want to add @puerco

File: contributors/contributors/core-committers.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ A core committer is a maintainer on the Mattermost project that has merge access
 
 Below is the list of core committers working on Mattermost:
 
-* [**Angelos Kyratzakos**](core-committers.md)
-  * @angelos.kyratzakos on [community.mattermost.com](https://community.mattermost.com/core/messages/@angelos.kyratzakos) and [@angeloskyratzakos](https://github.com/angeloskyratzakos) on GitHub
```
Comment (cpanato) [inline contributors/contributors/core-committers.md]: ~~why removing Angelos?~~

Comment (cpanato) [reply contributors/contributors/core-committers.md]: nvm :)

File: contributors/contributors/core-committers.md (hunk 1 above)
Comment (puerco) [reply contributors/contributors/core-committers.md]: That is more than enough, thank you!

Comment (justinegeffen) [inline contributors/contributors/core-committers.md]:
```suggestion
  * Dev areas: DevOps, SBOM, Containers, Kubernetes, Supply Chain
```

**Comment by cpanato** [comment]
@justinegeffen looks like there is a conflict, can you resolve that? thanks!

**Review by puerco — APPROVED** [review]
Thanks Justin, looks good to me!

**Pull request merged**